test/jvm/cast_test.rb

```diff
@@ -159,4 +159,32 @@ def bar(a:int[])
 
     assert_equal(2, cls.foo([2, 3].to_java(:int)))
   end
+  
+  def test_implisit_unboxing_array_iteration  
+     cls, = compile(<<-EOF)
+      def foo():int
+		list = [1,2,3]
+		m = 0
+		list.each do |x:int|
+			m += x
+		end
+		return m
```

Comment:
In the future, could you keep the same indentation as the rest of the code? We use spaces, 2x per level of indent.